src/protocol/cflie_nrf24l01.c

```diff
@@ -343,7 +343,7 @@ static void send_crtp_rpyt_packet()
         float pitch;
         float yaw;
         uint16_t thrust;
-    }__attribute__((packed)) cpkt;
+    }cpkt;
 
```

Comment:
why did you remove the packed attribute.  I don't know this protocol, but it is likely the packet is strictly defined as far as its structure goes, and the compiler can't insert any padding)

Reply:
there is risk for having packed structure on ARM platform as not ARM instruction support unaligned access. this get warned in the below code which can result unaligned access.

remove packed is not a right solution, i agree. let me tune the code a bit to manually pack the code to avoid this code accidently get broken by some compiler udpate.